modules/apps_api/lib/apps_api/notification_service.rb

```diff
@@ -46,17 +48,18 @@ def parse_event(event)
         app_record = DirectoryApplication.find_by(name: event['actor']['displayName'])
       end
       user = @okta_service.user(user_id)
-      create_hash(app_record: app_record, user: user, published: event['published'])
+      create_hash(app_record: app_record, user: user, published: event['published'], uuid: event['uuid'])
     end
 
-    def create_hash(app_record:, user:, published:)
+    def create_hash(app_record:, user:, published:, uuid:)
       {
+        'uuid' => uuid,
         'app_record' => app_record,
         'user_email' => user.body['profile']['email'],
         'options' => {
           'first_name' => user.body['profile']['firstName'],
           'application' => app_record ? app_record['name'] : nil,
-          'time' => published,
+          'time' => Time.zone.parse(published).strftime('%m/%d/%Y at %T:%M%p'),
```

Comment:
This could probably be extracted to a private method for readability. What does this time represent?

Reply:
➕

Reply:
extracted in latest commit